CHANGELOG.rst

```diff
@@ -11,6 +11,12 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+[0.9.0] - 2020-09-25
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Get rid of the ``documented_elsewhere`` annotation, which is used nowhere
```

Comment:
Not sure if we need to split deprecation and removal and use a major version change when we are ready? I always get burned when I try to cheat the system. Here are some references: https://github.com/search?q=documented_elsewhere&type=code

Reply:
AFAIK this is only used in edx-lint: https://github.com/search?q=org%3Aedx+documented_elsewhere&type=code

Here is the PR to get rid of it: https://github.com/edx/edx-lint/pull/127

Reply:
Sounds good.